src/main/java/codechicken/nei/ClientHandler.java

```diff
@@ -165,16 +154,19 @@ public static void loadSerialHandlers() {
         if (!file.exists()) {
             try (FileWriter writer = new FileWriter(file)) {
                 NEIClientConfig.logger.info("Creating default serial handlers list {}", file);
-                Collection<String> toSave = Loader.isModLoaded("dreamcraft") ? Collections.singletonList("")
-                        : Arrays.asList(defaultSerialHandlers);
-                IOUtils.writeLines(toSave, "\n", writer);
+                URL defaultSerialHandlersResource = ClientHandler.class
+                        .getResource("/assets/nei/cfg/serialhandlers.cfg");
+                if (defaultSerialHandlersResource != null) {
+                    IOUtils.copy(defaultSerialHandlersResource.openStream(), writer);
+                }
```

Comment:
There would be no file created, but that won't have any consequences other than NEI logging two error messages, no crashes or anything, it'll just continue with empty defaults. The reason for the null check is just to guard against an NPE to prevent an outright crash. However, I expect `getResource()` to never return null here, otherwise it seems like a lot of other things might be wrong as well.

Though you probably have more experience with that. It's only two more LOC anyway. Let me know if you still think it's worth it.

Reply:
Well, I was thinking, either we expect this case to ever happen, or we don't. If we expect it to happen, then we should gracefully handle it by creating an empty file; if we don't expect it to ever happen, we should just throw an exception.

But it probably doesn't matter too much either way

Reply:
OK, makes sense. I'll add the branch for consistency.

Reply:
Turns out, I was wrong above: if you use
```java
new FileWriter(PATH).close();
```
it automatically creates an empty file and since we are in a try-with-resources here, simply not doing any writes to the `writer` will create an empty file since `close()` will be called when exiting the scope. So the code already does what you want, no need for an else clause.